README.md

```diff
@@ -141,14 +153,14 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
 [kubeadm]: https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm/
 [design doc]: https://kind.sigs.k8s.io/docs/design/initial
 [user guide]: https://kind.sigs.k8s.io/docs/user/quick-start
-[SIG-Testing Mailing List]: https://groups.google.com/forum/#!forum/kubernetes-sig-testing
+[sig-testing mailing list]: https://groups.google.com/forum/#!forum/kubernetes-sig-testing
 [issue tracker]: https://github.com/kubernetes-sigs/kind/issues
 [filing an issue]: https://github.com/kubernetes-sigs/kind/issues/new
-[Kubernetes Slack]: http://slack.k8s.io/
+[kubernetes slack]: http://slack.k8s.io/
```

Comment:
can we please not alter things unrelated to the PR?
I'm also pretty sure this change is wrong, these are referenced in other places and the other places were not changed

Reply:
This issue has been resolved.